partner_event/views/event_registration_view.xml

```diff
@@ -1,31 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+<!-- © 2016 Tecnativa S.L. - Jairo Llopis
+     © 2016 Tecnativa S.L. - Vicent Cubells
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
-<openerp>
-<data>
+<odoo>
 
-<record id="view_registration_search" model="ir.ui.view">
-    <field name="name">Search attended registrations</field>
-    <field name="model">event.registration</field>
-    <field name="inherit_id" ref="event.view_registration_search"/>
-    <field name="arch" type="xml">
-        <!-- TODO Remove if https://github.com/odoo/odoo/pull/12997 is merged -->
-        <xpath expr="//filter[@domain=&quot;[('state','=','open')]&quot;]"
-               position="attributes">
-            <attribute name="name">open</attribute>
-        </xpath>
+    <record id="view_registration_search" model="ir.ui.view">
```

Comment:
I prefer to have that indentation level. That's not a guideline, but your preference. Not having this indentation level, IDEs don't work correctly with automatic indentations.